# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWindow.cpp
# Commit: 8a1503365add
# Full Hash: 8a1503365add26c02b96e92af35c919112b005f7
# Author: stransky <stransky@redhat.com>
# Date: 2022-07-26 21:44:38
# Regressor Bug: 1777664
# File Overlap Count: 1
# Description:
#   Bug 1777664 [Linux] Create nsWindow with initially paused compositor as we don't have valid GdkWindow/XWindow yet r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152688
# ==============================================================================

diff -r 396ad8e5d024 -r 8a1503365add widget/gtk/nsWindow.cpp
--- a/widget/gtk/nsWindow.cpp	Tue Jul 26 14:56:46 2022 +0000
+++ b/widget/gtk/nsWindow.cpp	Tue Jul 26 14:56:47 2022 +0000
@@ -5741,9 +5741,6 @@
   GtkWidget* container = moz_container_new();
   mContainer = MOZ_CONTAINER(container);
 
-  // Don't render to invisible window.
-  mCompositorState = COMPOSITOR_PAUSED_INITIALLY;
-
   // "csd" style is set when widget is realized so we need to call
   // it explicitly now.
   gtk_widget_realize(mShell);